enterprise_data/__init__.py

```diff
@@ -2,6 +2,6 @@
 Enterprise data api application. This Django app exposes API endpoints used by enterprises.
 """
 
-__version__ = "4.2.3"
+__version__ = "4.2.4"
```

Comment:
It seems that the version should be updated to 4.2.5 points or 4.2.5 logs in CHANGELOG merged with 4.2.4